src/main/java/seedu/address/model/commons/NameContainsKeywordsPredicate.java

```diff
@@ -4,22 +4,21 @@
 import java.util.function.Predicate;
 
 import seedu.address.commons.util.StringUtil;
-import seedu.address.model.activity.Activity;
 
 /**
- * Tests that a {@code Activity}'s {@code Name} matches any of the keywords given.
+ * Tests that a {@code TravelPlanObject}'s {@code Name} matches any of the keywords given.
```

Comment:
Name is also used by TravelPlan, not just TravelPlan object. Maybe we can add that in?